query-translator/tests/translate.rs

```diff
@@ -255,7 +306,7 @@ fn test_simple_or_join() {
                       [?page :page/title "Foo"])
                     [?page :page/url ?url]
                     [?page :page/description ?description]]"#;
-    let SQLQuery { sql, args } = translate(&schema, query, None);
```

Comment:
I was going to moan about all of these `None` instances... but they're deletions \o/